evalai/main.py

```diff
@@ -15,7 +15,7 @@ def main(ctx):
     """
```

Comment:
@isht3 I think we should also add the details about the `CHALLENGE` & `PHASE` keywords, so that the end user knows that we are referring to the challenge id and phase id here.